static/js/admin-scanner-query-result.js

```diff
@@ -0,0 +1 @@
+import './admin/scannerqueryresult';
```

Comment:
```suggestion
import './admin/scannerqueryresult.js';
```
?

Reply:
You don't need to specify the file extension. Vite will figure it out.

Reply:
oh, okay. The file extension was specified in all the other files. I don't mind whatever codestyle we settle on, but it'd be good to start consistently importing.

Reply:
Yes, I'll update the others. Vite will allow either. but right we should have consistency. This is also something we can enforce with eslint again once the migration is done.